lib/folio_record.rb

```diff
@@ -136,6 +163,28 @@ def items_and_holdings
     @items_and_holdings ||= client.items_and_holdings(instance_id:)
   end
 
+  def load_bound_with_parents
+    (record['boundWithParents'] ||
+      holdings.flat_map do |holding|
+        if holding&.dig('holdingsType', 'name') == 'Bound-with'
+          # API call returns relationship between a part of a bound-with (a holdings record) and the bound-with as a whole (the circulatable item)
+          bind = client.get_json("/inventory-storage/bound-with-parts?query=(holdingsRecordId==#{holding['id']})")
+          bind['boundWithParts'].map do |part|
+            parent = client.get_json("/inventory/items/#{part['itemId']}")
+            {
+              'childHoldingCallNumber' => holding['callNumber'],
+              'parentItemId' => parent['id'],
+              'parentItemBarcode' => parent['barcode'],
+              'parentInstanceTitle' => parent['title'],
+              # extract the instance id/ckey from the hrid to use later for displayed links
+              'parentInstanceHrid' => parent['hrid'].match(/\d+/)[0]
```

Comment:
For posterity, this won't work with newly created data in FOLIO or stuff from Lane (where the instance HRID will properly start with an L).. but maybe we cross that bridge if/when FOLIOs APIs are responsive enough that we can even consider using their APIs.

Reply:
@cbeer I am thinking I might just remove this whole part that does API calls until we are ready to use it.

Reply:
I think it's fine to keep it -- if nothing else, it's useful for comparing FOLIO's APIs to our direct database access